thorlcr/thorutil/thormisc.cpp

```diff
@@ -816,9 +816,9 @@ void setClusterGroup(INode *_masterNode, IGroup *_rawGroup, unsigned slavesPerNo
     IArrayOf<INode> clusterGroupNodes, nodeGroupNodes;
     clusterGroupNodes.append(*LINK(masterNode));
     nodeGroupNodes.append(*LINK(masterNode));
-    for (unsigned p=0; p<slavesPerNode; p++)
+    for (unsigned s=0; s<channelsPerSlave; s++)
```

Comment:
To match the script slave/port assigned order.
init_thor* starts the slavePerNode processes with a port differential of : channelsperslave * localthorportinc.
e.g. with slavesPerNode = 2 and channelsPerSlave = 2, slave process 1 starts at 20100, slave process 2 = 20500.

Prior to this change, the order with the clusterGroup (when looking up ip:port of a channel slave) was wrong.